### DISCUSSION TITLE
Welcome to opencrx-custom Discussions!

### DISCUSSION THREAD

**Discussion by opencrx** [opencrx/opencrx-custom#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we

**Comment by fschon** [comment]
Hi,

Firstly thanks for setting up this discussion area.

I have a few questions relating to segments please:

1. I read somewhere in the documentation that it is possible to move a segment from one server to another server. How can this be done?
2. Is there a query to delete an entire segment?
3. Is there a query to change the segment name?

Thanks,
Franz
